config/gflownet/trajectorybalance.yaml

```diff
@@ -8,3 +8,8 @@ optimizer:
   lr_decay_period: 1000000
   lr_decay_gamma: 0.5
 
+policy:
+  backward:
+    shared_weights: True
+    checkpoint: null
+    reload_ckpt: False
```

Comment:
If `shared_weights = False`, it should have `type`,  `n_hid` and `n_layers` as well. Do we want to add those here?